ding/utils/render_helper.py

```diff
@@ -0,0 +1,40 @@
+def render(env, render_mode=('rgb_array')):
+    '''
+    Overview:
+        Render the environment's current frame.
+    Arguments:
+        - env (:obj:`BaseEnv`): the ding env.
+        - render_mode (:obj:`str`): render mode.
+    Returns:
+        - frame (:obj:`numpy.ndarray`): [H * W * C]
+    '''
+    # TODO: do not use private member _env
+    if hasattr(env._env, 'sim'):
+        # mujoco
+        return env._env.sim.render(camera_name='track', height=128, width=128)[::-1]
```

Comment:
add some comments about why we need to reverse the result by `[::-1]`

Reply:
changed